check_limits.py

```diff
@@ -1,18 +1,36 @@
+battery_limits = {"temperature": [0, 45], "soc": [20, 80], "charge_rate": [0, 0.8]}
+					
+warn_limit = 5
 
-def battery_is_ok(temperature, soc, charge_rate):
-  if temperature < 0 or temperature > 45:
-    print('Temperature is out of range!')
-    return False
-  elif soc < 20 or soc > 80:
-    print('State of Charge is out of range!')
-    return False
-  elif charge_rate > 0.8:
-    print('Charge rate is out of range!')
-    return False
+lang_msg ={'DE' : {'warning' : "Warnung! Bitte halten Sie innerhalb des Bereichs",'Okay' : "Batterie ist in einwandfreiem Zustand", 'fail' : "Batterielimits A-Range!"},
```

Comment:
Any test possible to ensure that a translation is not missed when a maintainer adds something in future?